x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -503,7 +512,7 @@ export class RulesClient {
 
     // default duration of instance summary is 60 * alert interval
     const dateNow = new Date();
-    const durationMillis = parseDuration(alert.schedule.interval) * 60;
+    const durationMillis = parseDuration(alert.schedule.interval) * 360;
```

Comment:
I assume this was done for testing, and that it wouldn't be part of the PR.  When I re-looked at this, I was kind of shocked because I had been thinking the range was 6 * interval.  60 seems **waaay** too long!  Maybe 6 is too short though.  Seems like a potential nice size though.  Would be 30 minutes for a 5m interval, 6 days for a 24h interval.  Or we could just always make it 5 minutes or something.